app/tweep_dm.py

```diff
@@ -0,0 +1,239 @@
+                            "description": "Chemical & Electric Weapons (Tasers & Pepper Spray)"
+                        },
+                        {
+                            "label": "Rank 5",
+                            "metadata": "force_rank_5",
+                            "description": "Lethal Force (Guns & Explosives)"
+                        }
+                    ]
+                }
+            }
+        }
+    }
+]
+# Change to dict with keys of date location, and confirmation. Values list of A/B text
+list_of_A_B_txts = [
+    'Hi! I am a bot for Blue Witness, a project by @humanrights1st. We noticed your tweet may involve police misconduct, please confirm the date of this incident here: ',
+    'Hi! I am a bot for Blue Witness, a project by @humanrights1st. We noticed your tweet may involve police misconduct, please confirm the location of this incident here: ',
+]
+
+list_of_quick_replies = [quick_reply_use_of_force, confirmation_reply]
+
+
+def get_tweet_id(tweet_url):
+    """Get the tweet ID from the tweet URL"""
+    tweet_id = re.search(r'\d+$', tweet_url)
+    return tweet_id.group(0)
+
+
+def form_tweet(tweet_source: str, information_requested: str) -> Dict:
+    tweet_id = get_tweet_id(tweet_source)
+    if information_requested == 'date':
+        tweet_txt = list_of_A_B_txts[0]
+    if information_requested == 'location':
+        tweet_txt = list_of_A_B_txts[1]
+    link = os.getenv("FORM_URL")
+    reply_message = f"{tweet_txt} \n {link}"
+    tweet = api.update_status(reply_message, in_reply_to_status_id=tweet_id,
+                              auto_populate_reply_metadata=True)
+    return tweet
+
+
+def create_welcome_message(name: str,
+                           msg_txt: str,
+                           quick_replies: List[Dict] = None):
+
+    url = "https://api.twitter.com/1.1/direct_messages/welcome_messages/new.json"
+    if quick_replies:
+        payload = json.dumps({
+            "welcome_message": {
+                "name": name,
+                "message_data": {
+                    "text": msg_txt,
+                    "quick_reply": {
+                        "type": "options",
+                        "options": quick_replies
+                    }
+                }
+            }
+        })
+    else:
+        payload = json.dumps({
+            "welcome_message": {
+                "name": name,
+                "message_data": {
+                    "text": msg_txt
+                }
+            }
+        })
+    headers = {
+        'Content-Type': 'application/json'
+    }
+
+    response = manual_twitter_auth.post(url, headers=headers, data=payload)
+    print(response)
+    welcome_response = json.loads(response.text)
+    list_of_welcome_messages.append(welcome_response)
+    return list_of_welcome_messages
+
+
+# Can take out for loop for welcome_message by just using welcome_message_id
+# Have the welcome_message_name be displayed to admin but have it pass welcome_message_id
```

Comment:
Tomorrow I'll work with you and Brody on having the script selection functions in script_tracking.py replace some of this and clean it up the rest of the way.